### PR TITLE
mkosi: Add CI for CentOS Stream 10

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

**Review by bluca — APPROVED** [review]

**Comment by bluca** [comment]
Looks somewhat related?

```
[   65.822056] TEST-29-PORTABLE.sh[548]: + setfattr -n user.extension-release.strict -v false /tmp/wrongext/usr/lib/extension-release.d/extension-release.app0
[   65.833990] TEST-29-PORTABLE.sh[3063]: setfattr: /tmp/wrongext/usr/lib/extension-release.d/extension-release.app0: Operation not supported
```

**Comment by DaanDeMeyer** [comment]
It seems that doing user xattrs on tmpfs is a rather recent addition to the kernel (6.6) and I guess wasn't backported to the centos 9 kernel.

**Comment by bluca** [comment]
I see - pushed a fix, let's see if it works

**Comment by bluca** [comment]
```
+ exec /usr/lib/systemd/systemd-shutdown poweroff --timeout=90000000us --log-level debug,console:info --log-target=kmsg --exit-code=123 --exit-code=124
Failed to parse log level 'debug,console:info'. Ignoring.
Failed to parse log level debug,console:info, ignoring: Invalid argument
```

File: mkosi.postinst.chroot [hunk 1]
```diff
@@ -40,6 +40,10 @@ cp "$SRCDIR/factory/etc/nsswitch.conf" /etc/nsswitch.conf
 # Remove to make TEST-73-LOCALE pass on Ubuntu.
 rm -f /etc/default/keyboard
 
+# OpenSUSE disables all controller delegation for the user services. Remove the dropin to make TEST-55-OOMD
+# pass on OpenSUSE.
+rm -f /usr/lib/systemd/system/user@.service.d/20-defaults-SUSE.conf
```
Comment (bluca) [inline mkosi.postinst.chroot]: maybe mask the drop in in the test case instead? so that the generic image is more similar to a suse image

Comment (daandemeyer) [reply mkosi.postinst.chroot]: It's rather complicated to fix up the controllers after boot.

Comment (bluca) [reply mkosi.postinst.chroot]: What about command line/credential when booting for the test case?

Comment (daandemeyer) [reply mkosi.postinst.chroot]: We don't have a command line or credential for masking dropins AFAIK.

Comment (YHNdnzj) [reply mkosi.postinst.chroot]: You added support for passing arbitrary drop-ins through credentials, and passing an empty file should do the trick?

Comment (daandemeyer) [reply mkosi.postinst.chroot]: @YHNdnzj You think so? We need to symlink to /dev/null to mask a dropin, I don't think that logic is in place yet?

Comment (bluca) [reply mkosi.postinst.chroot]: masking can also be done with an empty file, doesn't need to be a symlink

**Review by bluca — APPROVED** [review]

**Pull request merged**